xml/System.Threading/Timer.xml

```diff
@@ -727,9 +727,9 @@ Sub TimerCallback(state As Object)
 ## Remarks  
  The callback method is invoked once after `dueTime` elapses, and thereafter each time the time interval specified by `period` elapses.  
   
- If `dueTime` is zero (0), the callback method is invoked immediately. If `dueTime` is negative one (-1) milliseconds, the callback method is never invoked; the timer is disabled, but can be re-enabled by calling <xref:System.Threading.Timer.Change%2A> and specifying a positive value for `dueTime`.  
+ If `dueTime` is <xref cref="F:System.TimeSpan.Zero?displayProperty=nameWithType" />, the callback method is invoked immediately. If `dueTime` is <see cref="F:System.Threading.Timeout.InfiniteTimeSpan" />, the callback method is never invoked; the timer is disabled, but can be re-enabled by calling <xref:System.Threading.Timer.Change%2A> and specifying a positive value for `dueTime`.  
```

Comment:
I didn't read all the comments, what is the difference between <xref> here and <see> above? If that's expected, we can merge it.

Reply:
This overload takes TimeSpans, so it's changing the text to use the named properties that are typed as TimeSpans and correspond to the cited values.